karma.shared.js

```diff
@@ -109,7 +109,14 @@
             logLevel: config.LOG_INFO,
 
             // Timeout for jasmine set to 50000 in TestHelpers.ExtendJasmineTimeout.js so warn if test time getting close
-            reportSlowerThan: 40000
+            reportSlowerThan: 40000,
+
+            // Settings to pass to frameworks and plugins
+            client: {
+                jasmine: {
+                    random: true
```

Comment:
This is the default, but just exposing in the config so it is easier to disable when debugging. It's actually a pretty good default, helps shake out bad infrastructure problems.